services/job_subscriber_test.go

```diff
@@ -112,6 +112,9 @@ func TestJobSubscriber_AddJob_Listening(t *testing.T) {
 		{"runlog v20190123 w/o address", "runlog", noAddr, newAddr(), 1, models.RunLogTopic20190123, cltest.StringToVersionedLogData20190123("id", `{"value":"100"}`)},
 		{"runlog v20190123 matching address", "runlog", sharedAddr, sharedAddr, 1, models.RunLogTopic20190123, cltest.StringToVersionedLogData20190123("id", `{"value":"100"}`)},
 		{"runlog w non-matching topic", "runlog", sharedAddr, sharedAddr, 0, common.Hash{}, cltest.StringToVersionedLogData20190123("id", `{"value":"100"}`)},
+		{"runlog v20190128 w/o address", "runlog", noAddr, newAddr(), 1, models.RunLogTopic20190123, cltest.StringToVersionedLogData20190128("id", `{"value":"100"}`)},
+		{"runlog v20190128 matching address", "runlog", sharedAddr, sharedAddr, 1, models.RunLogTopic20190123, cltest.StringToVersionedLogData20190128("id", `{"value":"100"}`)},
+		{"runlog w non-matching topic", "runlog", sharedAddr, sharedAddr, 0, common.Hash{}, cltest.StringToVersionedLogData20190128("id", `{"value":"100"}`)},
```

Comment:
Seems like this scenario is a duplicate of the case on line 114. ✂️

Reply:
It is a duplicate but with the data updated to 20190128 on line 117. But you know what, I need to update the topic for the two on lines 115 and 116. Getting that in now.